<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoedaCoin</class>
 <widget class="QMainWindow" name="MoedaCoin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoedaCoin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userMCCLabel">
        <property name="font">
         <font>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0.0 MCC</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="transactionTableWidget">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewWallet"/>
    <addaction name="actionOpenWallet"/>
    <addaction name="actionSaveWallet"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPublicKey"/>
   <addaction name="actionSendMoedacoin"/>
   <addaction name="actionMining"/>
  </widget>
  <action name="actionOpenWallet">
   <property name="text">
    <string>Open Wallet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveWallet">
   <property name="text">
    <string>Save Wallet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPublicKey">
   <property name="icon">
    <iconset resource="../res/moedacoin.qrc">
     <normaloff>:/icons/public-key-512x512.png</normaloff>:/icons/public-key-512x512.png</iconset>
   </property>
   <property name="text">
    <string>Public Key</string>
   </property>
   <property name="toolTip">
    <string>Visualize public key</string>
   </property>
  </action>
  <action name="actionMining">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/moedacoin.qrc">
     <normaloff>:/icons/minet-512x512.png</normaloff>:/icons/minet-512x512.png</iconset>
   </property>
   <property name="text">
    <string>Mining</string>
   </property>
   <property name="toolTip">
    <string>Mining Moedacoin</string>
   </property>
  </action>
  <action name="actionSendMoedacoin">
   <property name="icon">
    <iconset resource="../res/moedacoin.qrc">
     <normaloff>:/icons/send-coin-512x512.png</normaloff>:/icons/send-coin-512x512.png</iconset>
   </property>
   <property name="text">
    <string>Send Moedacoin</string>
   </property>
   <property name="toolTip">
    <string>Send Moedacoin to another user</string>
   </property>
  </action>
  <action name="actionNewWallet">
   <property name="text">
    <string>New Wallet</string>
   </property>
   <property name="toolTip">
    <string>Generate a pair of public and private keys for using in moedacoin</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/moedacoin.qrc"/>
 </resources>
 <connections/>
</ui>
